finality-aleph/src/block/substrate/mod.rs

```diff
@@ -76,9 +75,13 @@ impl BlockImporter {
 }
 
 impl BlockImport<Block> for BlockImporter {
-    fn import_block(&mut self, block: Block) {
-        let origin = BlockOrigin::NetworkBroadcast;
+    fn import_block(&mut self, block: Block, own: bool) {
+        let origin = match own {
+            true => BlockOrigin::Own,
+            false => BlockOrigin::NetworkBroadcast,
```

Comment:
If we leave this (see other comment), maybe we should comment why we consider all other cases or `BlockOrigin` to default to `NetworkBroadcast`?

Reply:
ok